packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -369,6 +369,12 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
 
   private messageReceivedListener(event: ChatMessageReceivedEvent): void {
+    const isCurrentChatAdapterThread = event.threadId === this.chatThreadClient.threadId;
+
+    if (!isCurrentChatAdapterThread) {
```

Comment:
nipick: do we even need the `const` in this case? I guess it makes it slightly clearer and easier to read. 
```suggestion
    if (event.threadId !== this.chatThreadClient.threadId) {
```

Reply:
I think using a descriptive variable name here could improve readability.